Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM centos:7
+ARG BINARY=./mobile-security-service-server
```

Comment:
Where is this binary coming from? In aerogear-app-metrics (on which this docker compose is based) there is a [makefile](https://github.com/aerogear/aerogear-app-metrics/blob/master/makefile) to build the required binary

Reply:
Apologies Damien, you need to generate a binary like so:

```sh
go build -o mobile-security-service-server cmd/mobile-security-service-server/main.go
```

We have not yet got a Makefile so this will have to done manually. There should be one added in the near future.

Reply:
@damienomurchu currently working on extending the README as we speak - https://issues.jboss.org/browse/AEROGEAR-8396

Reply:
@craicoverflow got ya - makes sense to include the README steps for getting docker-compose to work in this PR though?